financial-connections/src/main/java/com/stripe/android/financialconnections/features/common/ModalBottomSheetContent.kt

```diff
@@ -119,6 +122,20 @@ internal fun LegalDetailsBottomSheetContent(
     }
 }
 
+@Composable
+internal fun GenericBottomSheetContent(
+    generic: FinancialConnectionsGenericInfoScreen,
+    onClickableTextClick: (String) -> Unit,
+    onConfirmModalClick: () -> Unit
+) {
+    GenericScreen(
+        state = GenericScreenState(generic, inModal = true),
+        onPrimaryButtonClick = onConfirmModalClick,
+        onSecondaryButtonClick = { TODO("Secondary button click not yet implemented") },
```

Comment:
this drawer does not have a secondary button, but supporting this on a follow up PR.